draft-thomson-http-oblivious.md

```diff
@@ -908,7 +908,11 @@ encapsulated response.
 
 A proxy MUST NOT add information about the client identity when forwarding
 requests. This includes the Via field, the Forwarded field
-{{?FORWARDED=RFC7239}}, and any similar information.
+{{?FORWARDED=RFC7239}}, and any similar information.  A client does not depend
+on the proxy using an authenticated connection to the oblivious request
+resource, only that information about the client not be attached to forwarded
+requests.  A proxy SHOULD use TLS to protect requests that it forwards, as that
+can make some traffic analysis countermeasures taken by clients more effective.
```

Comment:
```suggestion
requests. 
```

Suggest dropping this, since it implies (to me) that one might want to run a target with HTTP support, which seems like an anti-goal, even if possible. I think the note here is sufficient.